src/Connection/ServiceConnection.ts

```diff
@@ -434,6 +443,32 @@ export class ServiceConnection {
 
         this.webSocket.send(message);
     };
+
+    // Function: RequestSessionStateChange
+    // Used to either start a new analytics session, or stop the current session.
+    RequestSessionStateChange = (
```

Comment:
Recommended rename to `RequestAnalyticsSessionStateChange` or a more general `AnalyticsSessionRequest` where the state variable (renamed to `ANALYTICS_SESSION_REQUEST` as recommended in other comment) decides the operation.